src/mapbox/poi.js

```diff
@@ -123,5 +124,15 @@ Poi.geocoderLoad = function(feature) {
   return poi
 }
 
+Poi.prototype.toUrl = function () {
+  let slug = ExtendedString.slug(this.name)
+  return `${this.id}@${slug}`
+}
+
+Poi.prototype.toAbsoluteUrl = function () {
+  let location = window.location
+  return `${location.protocol}//${location.host}/place/${this.toUrl()}/#map=${18}/${this.latLon.lat.toFixed(7)}/${this.latLon.lng.toFixed(7)}`
```

Comment:
This functions needs to take the "baseUrl" into account.  
Also, we could maybe read the "zoom" property instead of hardcoded "18" ?